test-harness/scenarios/rulesets/operation-security-defined.oas3.yaml

```diff
@@ -0,0 +1,3 @@
+extends: [[spectral:oas, off]]
+rules:
+  oas3-operation-security-defined: true
```

Comment:
@m-mohr In order to make the test more explicit (and resilient to potential future changes), how about setting `oas3-operation-security-defined` to `error` rather than just activating it with its default value?

Reply:
Makes sense, done.